packages/pcd-collection/src/PCDCollection.ts

```diff
@@ -10,9 +10,9 @@ export class PCDCollection {
   private packages: PCDPackage[];
   private pcds: PCD<any, any>[];
 
-  public constructor(packages: PCDPackage[], pcds: PCD[]) {
+  public constructor(packages: PCDPackage[], pcds?: PCD[]) {
```

Comment:
why make the second arg optional?

Reply:
just for ease of use, felt a bit silly to pass in an empty array